bigbluebutton-html5/private/config/settings.yml

```diff
@@ -472,13 +473,19 @@ public:
     typingIndicator:
       enabled: true
     moderatorChatEmphasized: true
-  note:
+  notes:
     enabled: true
-    url: ETHERPAD_HOST
+    id: notes
   layout:
     autoSwapLayout: false
     hidePresentation: false
     showParticipantsOnLogin: true
+  pads:
+    url: ETHERPAD_HOST
```

Comment:
Just drawing attention as this will trip some people. No change needed in the code, just commenting for reference
This is of the format `https://myurl.com/pad`